### PR TITLE
[IOTDB-3061] Support Show Child Paths/Nodes in Node Management

### PR DESCRIPTION
**Pull request by yifuzhou**
## Description
1confinode 1datanode demo:
image
image
image
image
image

All looks good.

### CODE REVIEW

**Comment by MarcosZyk** [comment]
It seems the queries on nodes below storage group work well. What about the nodes above storage group? For example, given sg ```root.test1.sg1``` and query ```show child paths root``` or ```show child paths root.test1```.

File: server/src/main/java/org/apache/iotdb/db/mpp/plan/planner/plan/node/metedata/read/ChildPathsSchemaScanNode.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  }
+
+  public static PlanNode deserialize(ByteBuffer buffer) {
+    String fullPath = ReadWriteIOUtils.readString(buffer);
+    PartialPath path;
+    try {
+      path = new PartialPath(fullPath);
+    } catch (IllegalPathException e) {
+      throw new IllegalArgumentException("Cannot deserialize ChildPathScanNode", e);
+    }
```
Comment (MarcosZyk) [inline server/src/main/java/org/apache/iotdb/db/mpp/plan/planner/plan/node/metedata/read/ChildPathsSchemaScanNode.java:70]: We can use the serialize method in PartialPath and deserialize method in PathDeserializeUtil, which can avoid the exception try-catch.

**Review by MarcosZyk — COMMENTED** [review]
PTAL

**Comment by MarcosZyk** [comment]
Please add some result screenshot about the cases across storage group.

**Review by MarcosZyk — APPROVED** [review]

**Comment by yifuzhou** [comment]
Currently this pr just supports node management below storage group. It needs some works from MPP site after discussion. Can merge this pr first and will adapt node management above storage group later.

**Review by SpriCoder — APPROVED** [review]
LGTM!

**Review by wangchao316 — APPROVED** [review]
LGTM

**Pull request merged**